src/main/java/com/amazon/opendistroforelasticsearch/knn/index/util/KNNConstants.java

```diff
@@ -21,4 +21,6 @@
     public static final String HNSW_ALGO_EF_CONSTRUCTION = "efConstruction";
     public static final String HNSW_ALGO_EF_SEARCH = "efSearch";
     public static final String HNSW_ALGO_INDEX_THREAD_QTY = "indexThreadQty";
+    public static final String L2 = "l2";
+    public static final String COSINESIMIL = "cosinesimil";
```

Comment:
If we do not support Cosine Similarity yet, remove from this PR

Reply:
Was adding incrementally in the PR should be available now.